aries-site/src/pages/components/card.mdx

```diff
@@ -71,32 +134,106 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 This type of card is more commonly used within marketing contexts and should be used sparingly within HPE products and services. Secondary purposes might include an action to navigate to a screen with greater detail on the offering.
```

Comment:
This paragraph might be nice to insert between "Primary purpose" and "Example use cases"

Reply:
Done.